src/containers/genePage/genomeFeatureWrapper.js

```diff
@@ -51,8 +51,7 @@ class GenomeFeatureWrapper extends Component {
     this.jbrowseUrl = externalJbrowseUrl;
   }
 
-
-  componentDidMount() {
+  componentDidUpdate() {
```

Comment:
The typical pattern is to make a data fetch call unconditionally in `componentDidMount` and conditionally based on checking `prevProps` in `componentDidUpdate`. See for example: https://github.com/alliance-genome/agr_ui/blob/f492f633334244bc9bf780ff3de83d58656ecc2f/src/containers/diseasePage/index.js#L34
As it stands this is firing off about half a dozen duplicate requests as the page loads which isn't desirable.

Reply:
I know.   Okay . . I'll give that a shot.

Reply:
I pushed the fix for this